docs/deployment.md

```diff
@@ -36,6 +36,24 @@ parameter_defaults:
   IronicInspectorInterface: 'br-ctlplane'
 ```
 
+## Ansible Networking Configuration
+
+If your deployment requires switch management, add/update these parameters in `standalone_parameters.yaml`:
+
+```
+parameter_defaults:
+  NeutronMechanismDrivers: [ansible, openvswitch, baremetal]
```

Comment:
Is order significant here? If it is, might be worth noting via a comment...

Reply:
Looks like it does! I'll add a note.